Tables/FE8 Nightmare modules/Class & Character editors/FE8 Character Editor.csv

```diff
@@ -144,7 +144,7 @@ Grado Remnant 1 (Main),0x24e,0x2aa,0x8d,0x9,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,
 Cutthroat,0x254,0x2ae,0x8e,0x41,0x0,0x6,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
 Grado Remnant 2 (Cutscene and C16 only?),0x24e,0x2aa,0x8f,0x5,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
 Grado Remnant 3 (C16 only?),0x24e,0x2aa,0x90,0x2d,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
-Grado Remnant 4 (C16 only?),0x24e,0x2aa,0x91,0x25,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
+Ch 26 generic,0x24e,0x2aa,0x91,0x25,0x0,0x3,0,0x0,1,10,0,5,0,-1,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
```

Comment:
I'd prefer if you just did this over 0xB5. That way we can keep each chapter's generics associated with their own, in case we decide to use it during polishing.